docusaurus/docs/ui-extensions/create-an-extension-point.md

```diff
@@ -15,40 +15,271 @@ keywords:
   - apps
 ---
 
-Providing an extension point within your UI enables other plugins to contribute supplementary capabilities, and leverage the context within the current view to guide users to a potential next step. Here's what you need to know about extension points.
+## What is an extension point?
```

Comment:
When the H1 (# Create an extension point) is added above this H2, it creates stacked headers which Grafana docs try to avoid. I suggest no H2  or adding a sentence above the H2 to introduce the topic as a whole.

Reply:
Thanks, that's a good point, I wasn't aware of this unfortunately 🙏 
I'll go with [Jack's suggestion](https://github.com/grafana/plugin-tools/pull/881#discussion_r1574263489) which adds a sentence above the H2, let me know what you think.